web/helpers.go

```diff
@@ -13,3 +14,10 @@ func StatusCodeForError(err interface{}) int {
 		return 500
 	}
 }
+
+// PublicError adds an error to the gin context and sets
+// the JSON value of errors.
+func PublicError(c *gin.Context, statusCode int, err error) {
```

Comment:
Why export this method?

Reply:
Don't have a good reason. I'll make it private.